src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MaxLatencySpec.groovy

```diff
@@ -242,6 +242,58 @@ class MaxLatencySpec extends HealthCheckSpecification {
         database.resetCosts()
     }
 
+    @Tidy
+    def "Able to create DEGRADED flow with LATENCY strategy if flowPath > max_latency"() {
+        given: "2 non-overlapping paths with 11 and 15 latency"
+        setLatencyForPaths(11, 15)
+
+        when: "Create a flow, maxLatency 5 and maxLatencyTier2 10"
+        def flow = flowHelperV2.randomFlow(switchPair).tap {
+            allocateProtectedPath = false
+            maxLatency = 9
+            maxLatencyTier2 = 16
+            pathComputationStrategy = PathComputationStrategy.LATENCY.toString()
+        }
+        northboundV2.addFlow(flow)
+
+        then: "Flow is created in DEGRADED state because flowPath doesn't satisfy max_latency value"
+        wait(WAIT_OFFSET) {
+            def flowInfo =  northboundV2.getFlow(flow.flowId)
+            assert flowInfo.status == FlowState.DEGRADED.toString()
+            assert flowInfo.statusInfo == "An alternative way (back up strategy or max_latency_tier2 value) of" +
+                    " building the path was used."
+        }
+        def path = northbound.getFlowPath(flow.flowId)
+        pathHelper.convert(path) == mainPath
+
+        cleanup:
+        flow && flowHelperV2.deleteFlow(flow.flowId)
+    }
+
+    @Tidy
+    @Tags([LOW_PRIORITY])
+    def "Able to create a flow with LATENCY strategy when max_latency > pathLatency > max_latency_tier2"() {
+        given: "2 non-overlapping paths with 11 and 15 latency"
+        setLatencyForPaths(11, 15)
+
+        when: "Create a flow, maxLatency 5 and maxLatencyTier2 10"
+        def flow = flowHelperV2.randomFlow(switchPair).tap {
+            allocateProtectedPath = false
+            maxLatency = 19
```

Comment:
12?

Reply:
There was an issue when we were not able to create a flow in case `max_latency > max_latency_tier2`
This test was added just to cover this case.